modules/fundamental/src/vulnerability/model/mod.rs

```diff
@@ -68,14 +68,18 @@ pub struct VulnerabilityHead {
 impl VulnerabilityHead {
     pub async fn from_vulnerability_entity(
         entity: &vulnerability::Model,
+        description: Option<Option<vulnerability_description::Model>>,
         tx: &ConnectionOrTransaction<'_>,
     ) -> Result<Self, Error> {
-        let description = entity
-            .find_related(vulnerability_description::Entity)
-            .one(tx)
-            .await?
-            .filter(|model| model.lang == "en")
-            .map(|model| model.description);
+        let description = match description {
+            Some(inner) => inner.map(|inner| inner.description),
+            None => entity
+                .find_related(vulnerability_description::Entity)
+                .filter(vulnerability_description::Column::Lang.eq("en"))
```

Comment:
random observation (more for my education) ... how far has localisation been considered throughout this app ?

Reply:
The API doesn't have anything inherently in any language. We support multiple languages for descriptions, as the various formats we consume kinda do also.

l10n for the UI? No idea.  @carlosthe19916